cmd/kubedirector/main.go

```diff
@@ -78,7 +68,20 @@ func main() {
 		validator.StartValidationServer(handler)
 	}()
 
-	sdk.Watch(resource, kind, namespace, resyncPeriod)
+	// The resync period essentially determines how granularly we can detect
+	// the completion of cluster config changes. Making this too small can
+	// actually be bad in that there is benefit to batch-resolving changes,
+	// within KubeDirector but also especially with the cluster's app config
+	// scripts. Using same resyncPeriod for all kinds that we will watch.
```

Comment:
Do we need a resyncPeriod at all for the config CR? We could set it to 0 so we don't get polling for that CR.

The only thing I wonder about in that case is whether we would stil get an initial handler when KD starts up and the config CR already exists. Hopefully so. (If not we could explicitly fetch it.)

Reply:
will test it out and let you know

Reply:
works as expected, when we reload kubedirector, handler does get called one time (if there is a CR)